tests/UnifiedSpecTests/UnifiedSpecTest.php

```diff
@@ -64,6 +59,15 @@ class UnifiedSpecTest extends FunctionalTestCase
         'valid-pass/entity-client-cmap-events: events are captured during an operation' => 'PHPC does not implement CMAP',
         'valid-pass/expectedEventsForClient-eventType: eventType can be set to command and cmap' => 'PHPC does not implement CMAP',
         'valid-pass/expectedEventsForClient-eventType: eventType defaults to command if unset' => 'PHPC does not implement CMAP',
+        // Command monitoring event serverConnectionId is not yet implemented
```

Comment:
No functional change here. Just shuffling around the entries and adding a comment above this group.